iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -252,6 +262,7 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
                     operator_manifest, pinned_by_iib, perform_sanity_checks=False
                 )
                 _apply_registry_replacements(bundle_metadata, registry_replacements)
+                _get_related_bundle_images(bundle_metadata, request_id)
```

Comment:
If this is accessing the related bundle images, it won't always work. I believe in the redhat-marketplace case, the new registries used do not yet contain the images.

Reply:
Correct, hence the decision to not overwrite the file if it already exists.

Reply:
Given that when this can be done depends on the customization for the organization, can we make this configurable? We could introduce a new type of customization that all it does is export the related bundle images. If not provided, it's always done at the end. Wdyt?

Reply:
That sounds like a good idea to me

Reply:
Thanks for taking this suggestion into account. I think the code looks much cleaner and contains less guess work.